.github/workflows/audit.yml

```
@@ -0,0 +1,19 @@
+name: Security Audit
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '59 23 * * *'
```

```suggestion
    - cron: '0 0 * * 0'
```

What about once a week ?

---

Sounds good to me :+1: